deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -208,3 +221,19 @@ podDisruptionBudget:
   # an integer (e.g. 1) or a percentage value (e.g. 25%).
   # Cannot be used if `minAvailable` is set.
   # maxUnavailable: 1
+
+# +docs:section=CRDs
+# The CRDs installed by this chart are annotated with "helm.sh/resource-policy: keep", this
+# prevents them from being accidentally removed by Helm when this chart is deleted. After
+# deleting the installed chart, the user still has to manually remove the remaining CRDs.
+crds:
+  # -- This option decides if the CRDs should be installed as part of the Helm
+  # installation.
+  enabled: false
```

Comment:
Not sure about this.
I can't work out how to allow the user to use venafi-connection when the VenafiConnection CRDs have already been installed via approver-policy-enterprise for example.